crates/bevy_winit/src/system.rs

```diff
@@ -81,6 +84,26 @@ pub(crate) fn create_windows<'a>(
                 window: window.clone(),
             });
 
+        #[cfg(target_arch = "wasm32")]
+        {
+            if window.fit_canvas_to_parent {
+                if let raw_window_handle::RawWindowHandle::Web(handle) =
+                    winit_window.raw_window_handle()
+                {
+                    let canvas: HtmlCanvasElement = web_sys::window()
+                        .unwrap()
+                        .document()
+                        .unwrap()
+                        .query_selector(&format!("canvas[data-raw-handle=\"{}\"]", handle.id))
+                        .unwrap()
+                        .unwrap()
+                        .unchecked_into();
```

Comment:
Is that too many unwraps ? We could use expects like https://github.com/gfx-rs/wgpu/blob/9f82504d8a3b6e234599b19cc35c299bcec8b138/wgpu-hal/src/gles/web.rs#L117

Reply:
Fortunately you don't need this whole block (and the corresponding crate features in `web-sys`), you can just use [`WindowExtWebSys::canvas()`](https://docs.rs/winit/0.29.9/wasm32-unknown-unknown/winit/window/struct.Window.html#method.canvas), which only needs one `unwrap()` that checks if you are on the main thread, which Bevy always is.